daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/engine/Result.scala

```diff
@@ -179,6 +179,9 @@ final case class ResultNeedUpgradeVerification[A](
 ) extends Result[A]
 
 object Result {
+
+  def done[A](a: A): Result[A] = ResultDone(a)
```

Comment:
is this ever used?

Reply:
I split the PR, it is a left over. 
I will remove it.